### PR TITLE
ROC-5768 request review of directions order reconsideration

### PR DESCRIPTION
**Pull request by robertparkinson**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-5768

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/app/claims/models/Direction.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { Moment } from 'moment'
+import * as _ from 'lodash'
+import { MomentFactory } from 'shared/momentFactory'
+
+export interface Direction {
```
Comment (KiranVarma02) [inline src/main/app/claims/models/Direction.ts]: Is there a better name? Direction seems very generic. is this directionOrder? but i see some otherdirectionOrder, so don't know what this is...

File: src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk [hunk 2]
```diff
@@ -331,3 +331,9 @@
 {% macro claimRejectedFreeMediationNOWithoutDQsForDefendantDashboard() %}
   {{ t('You’ve rejected the claim and suggested mediation. We’ll ask the claimant if they agree to take part in mediation.') }}
 {% endmacro %}
+
+
+
+
+
+
```
Comment (sabahirfan) [inline src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk]: can we please remove these additional empty lines.

File: src/main/app/claims/models/Direction.ts (hunk 1 above)
Comment (robertparkinson) [reply src/main/app/claims/models/Direction.ts]: each directions order can have multiple directions that need to be complied with. Directions might be a better name as it does return the array of directions

File: src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk (hunk 2 above)
Comment (robertparkinson) [reply src/main/features/dashboard/views/macro/claimStatus/claimRejected.njk]: done

**Pull request merged**